packages/beacon-node/src/util/dataColumns.ts

```diff
@@ -74,13 +75,17 @@ export class CustodyConfig {
   private config: ChainForkConfig;
   private nodeId: NodeId;
 
-  constructor(nodeId: NodeId, config: ChainForkConfig) {
+  private readonly metrics: Metrics | null;
+
+  constructor(nodeId: NodeId, config: ChainForkConfig, metrics: Metrics | null) {
```

Comment:
```suggestion
  constructor(nodeId: NodeId, config: ChainForkConfig, metrics: Metrics | null = null) {
```
this way you don't need to keep passing null

Reply:
slightly favor explicit approach here as it's less error pronse